ocrd/ocrd/processor/helpers.py

```diff
@@ -73,14 +80,19 @@ def run_processor(
     )
     log = getLogger('ocrd.processor.helpers.run_processor')
     log.debug("Running processor %s", processorClass)
-    processor = processorClass(
-        workspace,
+
+    processor = get_processor(
+        # TODO: Warning: processorClass of type Object gets auto casted to Type[Processor]
```

Comment:
How do you mean? Is the class "downgraded" to a `ocrd.Processor`?

Reply:
The comment is not relevant anymore if no typing is provided for the `processorClass`. I should have removed it. I was not sure if the auto-casting may potentially be problematic here since my IDE highlighted it as a potential problem.